app/presenters/statistics_announcement_presenter.rb

```diff
@@ -55,6 +57,14 @@ def page_title
     "#{super} - #{I18n.t("content_item.schema_name.#{document_type}", count: 1)}"
   end
 
+  def forthcoming_notice_title
+    "#{FORTHCOMING_NOTICE} #{release_date}"
+  end
+
+  def forthcoming_publication?
+    Time.zone.parse(content_item["details"]["display_date"]) > Time.zone.now && !cancelled?
```

Comment:
How does this handle the display_date: "November to December 2017" 
https://www.gov.uk/api/content/government/statistics/announcements/housing-statistics-1-april-2017-to-30-september-2017

Reply:
@fofr Quite right about the dates - I've updated the logic. 

The "reads a little oddly" - agreed. I've opened https://github.com/alphagov/government-frontend/pull/502 as a starter to discuss improving the clarity of the wording.